fdbserver/storageserver.actor.cpp

```diff
@@ -7569,6 +7573,7 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 			}
 
 			state Key nfk = keys.begin;
+			state Key blockBegin = keys.begin;
```

Comment:
Currently, they're the same, which basically equals `keyAfter(block.end().key`. But not sure if it's bug that readThrough not set, if it's set, then seems it's going to be [`block.end().key`](https://github.com/apple/foundationdb/blob/117b6c0a6ea35ad8e43740ac201e6c0e0ad6becf/fdbserver/VersionedBTree.actor.cpp#L8264) (instead of `keyAfter()`).
Anyway, I'll reuse nfk here and rename it to blockBegin. (I guess `nfk` means `nextFetchKey`?)